fsw/src/to_lab_app.c

```diff
@@ -68,25 +67,19 @@ TO_LAB_GlobalData_t TO_LAB_Global;
 /*
 ** Include the TO subscription table
 **  This header is in the platform include directory
-**  and can be changed for default TO subscriptions in 
+**  and can be changed for default TO subscriptions in
 **  each CPU.
 */
 #include "to_lab_sub_table.h"
 
 /*
 ** Event Filter Table
 */
-static CFE_EVS_BinFilter_t  CFE_TO_EVS_Filters[] =
-          { /* Event ID    mask */
-            {TO_INIT_INF_EID,           0x0000},
-            {TO_CRCMDPIPE_ERR_EID,      0x0000},
-            {TO_SUBSCRIBE_ERR_EID,      0x0000},
-            {TO_TLMOUTSOCKET_ERR_EID,   0x0000},
-            {TO_TLMOUTSTOP_ERR_EID,     0x0000},
-            {TO_MSGID_ERR_EID,          0x0000},
-            {TO_FNCODE_ERR_EID,         0x0000},
-            {TO_NOOP_INF_EID,           0x0000}
-          };
+static CFE_EVS_BinFilter_t CFE_TO_EVS_Filters[] = {/* Event ID    mask */
+                                                   {TO_INIT_INF_EID, 0x0000},       {TO_CRCMDPIPE_ERR_EID, 0x0000},
```

Comment:
This change looks funky

Reply:
Common pattern w/ this format style.  I'd argue personal preference, either way (orig or formatted) is readable.

Reply:
There are a lot of changes this tool does that look "funky" ... maybe preference, but definitely some which make it less readable/obvious.  One entry per line is definitely more readable than what the tool produced.

Reply:
I like the one entry per line as well but I can live with this if there's no easy way to make the tool use something like the original format.

Reply:
I encourage anyone who feels strongly enough about it to investigate alternative configuration settings and propose them.  I'd prefer not to hold up all the format benefits the current configuration provides though, we can always just apply a change in the future if one is agreed to.

Reply:
Agreed, we'll merge as is and improve as we go.